libraries/erlang_install.rb

```diff
@@ -0,0 +1,83 @@
+#
+# Cookbook Name:: languages
+# HWRP:: erlang_install
+#
+# Copyright 2015, Chef Software, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+class Chef
+  class Resource::ErlangInstall < Resource::LWRPBase
+    resource_name :erlang_install
+
+    actions :install
+    default_action :install
+
+    attribute :version,  kind_of: String, name_attribute: true
+    attribute :checksum, kind_of: String, required: true
```

Comment:
So we should just update `remote_install` to the `checksum` attribute optional. No reason to leak that impl detail on this resource.